olp-cpp-sdk-core/include/olp/core/generated/serializer/SerializerWrapper.h

```diff
@@ -24,8 +24,8 @@
 #include <string>
 #include <vector>
 
-#include <boost/optional.hpp>
 #include <rapidjson/document.h>
+#include <boost/optional.hpp>
```

Comment:
This sorting seems to be not alphabetically. Is there any other sorting?

Reply:
clang-format, I'll revert it manually